web/gds-user-ui/src/components/CertificateManagement/MainnetTestnetCertificates.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  Stack,
+  Box,
+  Table,
+  TableCaption,
+  Heading,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+  Badge,
+  IconButton
+} from '@chakra-ui/react';
+import { t, Trans } from '@lingui/macro';
+import ConfirmIdentityCertificateModal from 'components/ConfirmIdentityCertificateRequest';
+import CkLazyLoadImage from 'components/LazyImage';
+import { NoData } from 'components/NoData';
+import FormLayout from 'layouts/FormLayout';
+import { Certificate } from 'types/type';
+import Statistics from './Statistics';
+import PolygonIcon from 'assets/polygon.svg';
+import dayjs from 'dayjs';
+import { useNavigate } from 'react-router-dom';
+
+type MainnetCertificatesProps = {
+  data: Certificate[];
+  network: 'mainnet' | 'testnet';
+};
+
+const DATE_FORMAT = 'DD-MM-YYYY';
+
+function MainnetTestnetCertificates({ network, data }: MainnetCertificatesProps) {
+  const navigate = useNavigate();
+  const handleDetailsClick = (certificateId: string) => {
+    navigate(`/dashboard/certificate-inventory/${certificateId}?network=${network}`);
+  };
+
+  return (
+    <Stack spacing={5}>
+      <Statistics />
+      <Box>
+        <FormLayout overflowX={'scroll'}>
+          <Table variant="simple">
+            <TableCaption placement="top" textAlign="start" p={0} m={0}>
+              <Stack direction={'row'} alignItems={'center'} justifyContent={'space-between'}>
+                <Heading fontSize={'1.2rem'} data-testid="title">
+                  {network === 'mainnet' ? (
+                    <Trans>MainNet Identity Certificates</Trans>
+                  ) : (
+                    <Trans>TestNet Identity Certificates</Trans>
+                  )}
+                </Heading>
+                <ConfirmIdentityCertificateModal marginLeft="auto !important">
+                  <Trans>Request New Identity Certificate</Trans>
+                </ConfirmIdentityCertificateModal>
+              </Stack>
+            </TableCaption>
+            <Thead>
+              <Tr>
+                <Th>
+                  <Trans>Serial No</Trans>
+                </Th>
+                <Th>
+                  <Trans>Issue Date</Trans>
+                </Th>
+                <Th>
+                  <Trans>Expiration Date</Trans>
+                </Th>
+                <Th>
+                  <Trans>Status</Trans>
+                </Th>
+                <Th textAlign="center">
+                  <Trans>Actions</Trans>
+                </Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              {data && data.length ? (
+                data?.map((certificate, idx) => (
+                  <Tr key={idx} data-testid="table-row">
+                    <Td>{certificate.serial_number}</Td>
+                    <Td>{dayjs(certificate.issued_at).format(DATE_FORMAT)}</Td>
```

Comment:
it's possible to create an abstraction of this date format? an external function like you did for `formatDisplayedValue`.